index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="ru">
+	<head>
+		<meta charset="UTF-8">
+		<title>Dzianis Kavalionak Resume</title>
+		<link rel="stylesheet" href="main.css">
+	</head>
+	<body>
+		<script src="zeros.js"></script>
+		<header>
+			<img class="header_bg" src="img/bg_photo.jpg" alt="s" width="1024" height="500">
```

Comment:
`html/css`: You don't have to set 1024 manually everywhere, you can just make container (`<header>`) with width 1024px, position relative, and all children can be width: 100%.